src/agent0/core/hyperdrive/agent/hyperdrive_actions.py

```diff
@@ -167,8 +167,8 @@ def close_short_trade(
 
 def add_liquidity_trade(
     trade_amount: FixedPoint,
-    min_apr: FixedPoint = FixedPoint(scaled_value=1),
-    max_apr: FixedPoint = FixedPoint(scaled_value=2**256 - 1),
+    min_apr: FixedPoint | None = None,
```

Comment:
is this necessary? we made FixedPoint immutable, right? so the `Optional` syntax is not needed because there is no risk of modifying the variable passed in?

Reply:
No, this isn't necessary, but it's nice if the caller wants to pass in `None` for a default instead of either defining its own defaults or case switches for these optional values. It also matches how we're doing default values throughout the code

Reply:
we discussed; all good now.